docs/product-manuals/clients/go-client/get-started.md

```diff
@@ -46,9 +47,9 @@ export ZEEBE_CLIENT_SECRET='[Client Secret]'
 export ZEEBE_AUTHORIZATION_SERVER_URL='[OAuth API]'
 ```
 
-**Hint:** When you create client credentials in Camunda Cloud you have the option to download a file with the lines above filled out for you.
+**Hint:** When you create client credentials in Camunda Cloud, you have the option to download a file with the lines above filled out for you.
```

Comment:
Similar comment about admonitions, do you think we need something stronger here?

Reply:
Agreed. See my note above in the first comment regarding admonitions.